### DISCUSSION TITLE
Blank image generated when creating a tiled watermark

### DISCUSSION THREAD

**Discussion by oakif** [ImageMagick/ImageMagick#7429] [Help] [answered]
### ImageMagick version

7.1.1-34

### Operating system, version and so on

macOS 14 Sonoma

### Description

I am trying to create a tiled watermark using ImageMagick. The command I’m using is:

```
magick -background None -fill "#000000" -font Arial -kerning 0 -pointsize 100 label:"watermark" -rotate -45 +repage -write mpr:watermark +delete -size 1000x1000 xc:none tile:mpr:watermark watermark.png
```

This command results in two output files:

1. `watermark-0.png` – A blank 1000x1000 image.
2. `watermark-1.png` – The expected tiled watermark image, but with a different filename than specified.

I expected the output to be a single file named `watermark.png` containing the tiled watermark.

Could someone help me understand why this is happening and how I can modify the command to produce only the desired watermark file? Thanks for the assistance!

**Comment by snibgo** [comment, marked as answer]
Why do you have `xc:none`? If you remove it, does that do what you want?
